target/linux/generic/files/drivers/net/phy/rtl8367.c

```diff
@@ -1722,6 +1723,13 @@ static int rtl8367_detect(struct rtl8366_smi *smi)
 	dev_info(smi->parent, "RTL%s ver. %u chip found\n",
 		 chip_name, rtl_ver & RTL8367_RTL_VER_MASK);
 
+	if (of_get_property(smi->parent->of_node, "realtek,extif1", &size))
+		smi->cpu_port = RTL8367_CPU_PORT_NUM - 1;
+	else
+		smi->cpu_port = RTL8367_CPU_PORT_NUM;
```

Comment:
```
	if (of_get_property(smi->parent->of_node, "realtek,extif1", &size))
		smi->cpu_port -= 1;
```

Reply:
At the time of calling probe(), we do not know the chip version yet.